tests/integration/calendar/test_calendar_props.py

```diff
@@ -42,6 +42,8 @@ def test_cdpr001_date_clearable_true_works(dash_dcc):
     (single_date,) = dash_dcc.get_date_range("dps")
     assert not single_date, "date should be cleared"
 
+    assert dash_dcc.get_logs() == []
```

Comment:
helpful message on assertion failure.

Reply:
This is a pattern we use at the end of most tests - it just means "there are no errors in the JS console"

We could probably make it clearer to read though by building this directly into the fixture, like:
```py
dash_duo.assert_clean_console()
```
(`dash_dcc` is a `dash_duo` clone with some dcc-specific additions. I say clone because it behaves like a subclass but isn't really... the subclassing happens one layer down.)